robyn/robyn.pyi

```diff
@@ -177,6 +177,8 @@ class QueryParams:
         pass
 
 class Headers:
+    exclude_paths: Optional[list[str]] = None
```

Comment:
Yes, headers should not have an `exclude_paths` method. This is not a great design. You need to figure out a way to store this somewhere else

Reply:
@sansyrox how about adding it to the `Server` struct? just like `global_request_headers`, & `global_response_headers`?

Reply:
@VishnuSanal , that should be a better place. But the data structure should not be super general.